roles/local-path-provisioner/tasks/main.yml

```diff
@@ -17,27 +17,21 @@
 - name: Include platform-specific common tasks
   include_tasks: "_{{ ansible_os_family | lower }}.yml"
 
-- name: Checkout baremetal repo
+- name: Checkout local-path-provisioner repo
   include_role:
     name: baremetal
-    tasks_from: checkout_baremetal_repo
-    
-- name: Organize charts repo
-  include_role:
-    name: baremetal
-    tasks_from: organize_charts_repo_infra
-
-- name: Create powerdns_helm_overrides.yml
-  template:
-    src: powerdns_helm_overrides.yml.j2
-    dest: ~/powerdns_helm_overrides.yml
+    tasks_from: checkout_repo
+  vars:
+    repo_dir: local-path-provisioner
+    repo_url: "{{ local_path_provisioner_repo_url }}"
+    repo_branch: "{{ local_path_provisioner_branch }}"
+    repo_pull_request: "{{ local_path_provisioner_pull_request | default('') }}"
```

Comment:
Do we need to check this chart out from source, or could we use e.g `kubernetes.core.helm_repository` similar to MetalLB (I know this isn't handled consistently elsewhere, I'm just thinking that moving away from source-checkouts may simplify things?)

Reply:
There is no chart repo in the `local-path-provisioner` (index.yaml) file, and we cannot install the chart directly because it is not hosted anywhere. That's why I clone the repo

Reply:
Ack thanks for clarifying 👍